### PR TITLE
Update Platforms pages to include ARE

### PR DESCRIPTION
**Pull request by hot007**
Update platforms pages to fix reference names, and importantly, add information on NCI's new ARE offering. 

Requested a few reviews but I'm sure we don't need all 4, just tagging you in case you want to review!

### CODE REVIEW

File: BigData/platforms/platforms-nci-gadi.md [hunk 1]
```diff
@@ -1,28 +1,41 @@
...
 - the `MDSS` mass datastore facility for data archiving/backup.
 - help via help@nci.org.au as well as specialised climate support from CLEX, BoM and CSIRO support staff via cws-help@nci.org.au.
 
+The HPC is usually accessed via the command line to login nodes, which support internet access. Tasks to be run on the HPC compute nodes are described in "job scripts" which are submitted to a batch queuing system to ensure fair use of the resources. No internet access is available from Gadi compute nodes, so all downloads/repository cloning needs to be done on login nodes.
+
+The NCI also support a virtual desktop infrastructure (VDI) and JupyterLab via the Australian Research Environment (ARE), provides an interface to the HPC infrastructure (currently limited to broadwell CPU and GPU nodes). The ARE provides a web-based interface to Gadi, a virtual desktop on a compute node, or a JupyterLab session. Internet access is not available on broadwell queue nodes so the VDI cannot be used for downloading or external `git` work but is excellent for dask-based data analysis. Documentation can be [found here](https://opus.nci.org.au/display/Help/ARE+User+Guide).
```
Comment (dougiesquire) [inline BigData/platforms/platforms-nci-gadi.md:15]: Some repetition I think. Maybe:
"The NCI also support an interface to the HPC infrastructure called the Australian Research Environment (ARE). The ARE provides a web-based interface to Gadi via a virtual desktop infrastructure (VDI) on a compute node, or a JupyterLab session (both are currently limited to broadwell CPU and GPU nodes). Internet access is not available on broadwell queue nodes so the ARE cannot be used for downloading or external `git` work but is excellent for dask-based data analysis. Documentation can be [found here](https://opus.nci.org.au/display/Help/ARE+User+Guide)."

File: BigData/platforms/platforms-nci-gadi.md [hunk 2]
```diff
@@ -1,28 +1,41 @@
...
 - NCI is an ideal platform to use when collaborating with peers from other institutions as it is available to all Australian researchers. 
 - If you are wanting to use large reference climate datasets as input for your research, chances are the data you want may already be available at NCI, which saves you time downloading and managing the datasets, and it is more efficient for everyone to use these centralised collections.
 - If you need to work interactively using code in your Gadi `/home` directory instead of the [OOD](https://acdguide.github.io/BigData/platforms/platforms-nci-ood.html), there are [helper scripts](https://github.com/coecms/nci_scripts) such as `gadi_jupyter` provided by the CLEX CMS team.
```
Comment (dougiesquire) [inline BigData/platforms/platforms-nci-gadi.md:27]: I wonder if we want to add your point below to the end of this point, which mentions the OOD, e.g. something like:
"Note, however, that the recently launched ARE provides a JupyterLab environment with access to Gadi `/home` and is the recommended tool for users looking to perform interactive large-scale data analysis"

File: BigData/platforms/platforms-nci-ood.md [hunk 3]
```diff
@@ -1,6 +1,8 @@
-(ood)=
+(OOD)=
 ## NCI OOD Virtual Desktop Infrastructure
 
+**NOTE that NCI have recently launced their ARE platform, which may ultimately replace this OOD offerring, this information is current as of July 2022.**
```
Comment (dougiesquire) [inline BigData/platforms/platforms-nci-ood.md:4]: Typo in offering

**Review by dougiesquire — APPROVED** [review]
Thanks @hot007. Some very minor comments to take or leave

File: BigData/platforms/platforms-nci-gadi.md (hunk 1 above)
Comment (hot007) [reply BigData/platforms/platforms-nci-gadi.md:15]: Slightly refactored into 3 short paragraphs to separately address HPC via terminal, ARE, internet access.

File: BigData/platforms/platforms-nci-gadi.md (hunk 2 above)
Comment (hot007) [reply BigData/platforms/platforms-nci-gadi.md:27]: Opted to add to the dot points below, but good point, the ARE needs to be listed before the CLEX scripts now.

File: BigData/platforms/platforms-nci-ood.md (hunk 3 above)
Comment (hot007) [reply BigData/platforms/platforms-nci-ood.md:4]: Done

**Comment by hot007** [comment]
@paolap I think I fixed the headers to be the correct indent level (a heap of conflicts had appeared), it'd be good to get this one merged if you could check please?

**Comment by paolap** [comment]
I just built this and there are some header warnings but not in this section, mostly in format_metadata.md which might have already been taken care of separately. In any case they shouldn't stop you from merging

**Comment by paolap** [comment]
Sorry I did miss one warning which needs fixing and that is the platform-BoM.md file still has `##` instead of `#` so it doesn't recognise the header as a title

**Comment by hot007** [comment]
Whoops thanks - fixed BoM file.

**Review by paolap — APPROVED** [review]
On least minor comment in the file Platform-PC.md
change the link  
"See the [Tools](https://acdguide.github.io/BigData/different_tools.html) page"
to a direct reference to the file:
[Tools/Software](../tools/tools-intro.md) or whatever page you want to link. NB We decided to rename tools to software.
 Should the filename change jupyter will warn you when you build the book, but using the url might fail silently.

**Comment by hot007** [comment]
Good pick up, thanks @paolap . Fixed that and I'm going to go ahead and merge.

**Pull request merged**